doc/tutorial/configuring-stylesheet.rst

```diff
@@ -0,0 +1,43 @@
+Working with the stylesheet
+===========================
+
+To recap what we've built so far:
+
+* a Sphinx project containing some content across multiple sections
+* a PDF version of the output, using rinohtype's Sphinx builder, alongside
+  Sphinx's built-in HTML rendering
+* a local template configuration file, ``my-article.rtt``, that is invoked in
+  ``conf.py``, and tells rinotype to:
```

Comment:
You missed the _h_ in rinohtype.

Actually, I'm not so enthusiastic about the project's name. It's too easy to confuse with "rhino" and just doesn't sound snappy. If you have any suggestions for a better name, don't hesitate to share! 😄